enzyme/test/Integration/ReverseMode/stl_umap.cpp

```diff
@@ -0,0 +1,82 @@
+// RUN: %clang++ -std=c++11 -fno-exceptions -ffast-math -O0 %s -S -emit-llvm -o - | %opt - %OPloadEnzyme %enzyme -S | %lli -
```

Comment:
`-O0` fails to compile even for `operator[]`, likely because it strumbles over the map insert (which would be good to support in more general cases, too):
https://fwd.gymni.ch/R6Q2bQ